README.md

```diff
@@ -4,32 +4,67 @@
 
-A JavaScript file that powers a bookmarklet to summarize some time-keeping helpers.
+A link that you can add to your browser's bookmarks that adds a summary and some time-keeping helpers to your unanet timesheet page.
 
 ![screenshot of the tool in action](bookmarklet/summarizer-screenshot.png)
 
-## Get the Bookmarklet
+## For Users
 
-Drag this link to your browser's bookmarks or bookmark toolbar (you can then rename the title to what you want)
+### Purpose
+
+We (and our supervisees) sometimes have a lot of time entry categories. It may be difficult to understand a high-level view of these entries at a glance.
+
+The Unanet Summarizer allows you to:
+
+* View your summary of time entries by time (INT, CORE+, BILL+, etc.)
+* See how "over/under" you are for a given time period
+  * ...and carry over the first timesheet's number to your second timesheet so you can see for the month
+* Avoid using more PTO than you need to by seeing if you're over on `+` hours in a time period where you use PTO
+* Quickly summarize your supervisees' timesheets to see where their hours land (and watch for signs of overwork/burnout, etc.)
+
+### How to Get the Bookmarklet
+
+Drag the link below to your browser's bookmarks or bookmark toolbar (you can rename it afterward, if you want)
 
 <a href="javascript:(function(){ var onload=function(){ window.summarizeUnanetTime(); }; if(!window.summarizeUnanetTime){var s=document.createElement('script');s.src='https://excellalabs.github.io/unanet-summarizer/unanet-summarizer.js';s.onload=onload;document.body.appendChild(s)} else { onload(); } })();">Summarize Unanet</a>
 
-## How the tool works
+Here's an animation of how that might work:
+
+> ![Demo of dragging the link to bookmarks](images/demo_adding-bookmarklet.gif)
+
+### How to Use the Bookmarklet
+
+When you're on your timesheet (or viewihng your supervisees' timesheet), click the bookmarklet link to see the time summary.
+
+> ![Demo of using the bookmarklet on a timesheet](images/demo_adding-bookmarklet.gif)
+
+### Common Questions
+
+#### Q: What's a Bookmarklet?
+
+A: Most bookmarks point to a web site (e.g. <https://excellalabs.com>). A _bookmarklet_ actually contains some script that manipulates a web page in some way. In our case, that script downloads the unanet-summarizer code and runs it, which displays the summary box on the web page.
+
+#### Q: What do you mean by my browser's bookmarks toolbar?
+
+A: On many browsers, you can display a "bookmarks bar" below the main place where you enter web sites / searches. This lets you click on a bookmark directly, without going through your "bookmarks" or "favorites" folders. It makes things a little more accessible.
```

Comment:
Great question/answer, the second one specifically. On that and FWIW: I feel that Apple really muddied the water when they created Favorites (the tiles that show up when you open a new tab in Safari) which got implemented alongside Bookmarks. In fact I think I may still be unclear on some aspects of where those two overlap, if they do at all. If you think that's a common confusion, then perhaps consider the start of this question might be a good place to knock that out right off the bat, aka "We're not talking about Favorites..." - or could be a separate question altogether before this one, just to level set.

Reply:
I'd probably consider this particular change as part of another PR since that's the first time I've heard of this question and nobody's asked anything similar in a few years.

BUT, that does remind me that we should probably include something to ensure people know where they can ask questions.